.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
```

Comment:
How about a leading `Overivew` or `Problem Statement` section?

Reply:
What should we do as subsection/example there?

Reply:
Isn't that pretty much what the question below is?